perfmetrics/scripts/fio/install_fio.sh

```diff
@@ -32,10 +32,17 @@ echo "Installing fio ..."
 sudo apt-get install -y libaio-dev
 
 # We are building fio from source because of the issue: https://github.com/axboe/fio/issues/1668.
+# The sed command below is to address internal bug#309563824. 
+# As recorded in this bug, fio by-default supports 
+# clat percentile values to be calculated accurately upto only 
+# 2^(FIO_IO_U_PLAT_GROUP_NR + 5) ns = 14 ms 
```

Comment:
nit: I think it's 17.17 ms not 14ms.

Reply:
Also, it would be good to give fio_issue link in the comment.

Reply:
@raj-prince Corrected to 17.17 and 137.44 seconds.

The fio issue link is there in the first line of comment. `We are building fio from source because of the issue: https://github.com/axboe/fio/issues/1668`